client/foundries.go

```diff
@@ -862,6 +862,17 @@ func (a *Api) FactoryListDeviceGroup(factory string) (*[]DeviceGroup, error) {
 	return &resp.Groups, nil
 }
 
+func (a *Api) CiTargets(factory string) (*AtsKey, error) {
```

Comment:
Can you rename this into smth like `GetOnlineTargetsKey` ?
As is I constantly have a feeling that you fetch targets when calling this method.

Reply:
https://github.com/foundriesio/fioctl/pull/84/commits/9b5c948da0ef274d59c51c22bc3173f21bf18046